README.md

```diff
@@ -730,6 +730,19 @@ Name | Type | Required | Description | Example
 --- | --- | --- | --- | ---
 remotePath | string | yes | The full path to the remote file or a file inside an application bundle | `/sdcard/myfile.txt` or `@my.app.id/path/in/bundle`
 
+### mobile: startActivity
+
+Starts the given activity with intent options, activity options and locale.
```

Comment:
it makes sense to explicitly mention that the activity could only be executed in scope of the current app package

Reply:
Added note about appPackage.